infrastructure/ithc.tfvars

```diff
@@ -0,0 +1,8 @@
+external_host_name = "gateway-ccd.ithc.platform.hmcts.net"
+cors_origin = "*"
```

Comment:
Ideally this should be set to the new DNS entries.  But lets leave that for once we have a working env.